vm-docker/Dockerfile

```diff
@@ -0,0 +1,117 @@
+FROM 'google/cloud-sdk:slim'
+
+# install packages 
+RUN apt-get update && apt-get install -y \
+    apt-utils \
+    default-jdk \
+    g++ \
+    less \
+    libbz2-dev \
+    libcurl4-openssl-dev \
+    liblzma-dev \
+    libncurses5-dev \
+    pkg-config \
+    python3-venv \
+    unzip \
+    wget \
+    zlib1g-dev \
+    && \
+	# clean up apt cache
+    rm -rf /var/lib/apt/lists/*
+
+RUN mkdir home/tools 
+
+# Install htslib
+ENV SAMTOOLS_VERSION="1.11"
+RUN cd home/tools \
+    && wget https://github.com/samtools/htslib/releases/download/${SAMTOOLS_VERSION}/htslib-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& tar xjf htslib-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& rm htslib-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& cd htslib-${SAMTOOLS_VERSION} \
+	&& ./configure \
+	&& make \
+	&& make install \
+	&& make clean
+
+
+# Install bcftools
+RUN cd home/tools \
+    && wget https://github.com/samtools/bcftools/releases/download/${SAMTOOLS_VERSION}/bcftools-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& tar xjf bcftools-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& rm bcftools-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& cd bcftools-${SAMTOOLS_VERSION} \
+	&& ./configure \
+	&& make \
+	&& make install \
+	&& make clean
+
+# Install samtools
+RUN cd home/tools \
+    && wget https://github.com/samtools/samtools/releases/download/${SAMTOOLS_VERSION}/samtools-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& tar xjf samtools-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& rm samtools-${SAMTOOLS_VERSION}.tar.bz2 \
+	&& cd samtools-${SAMTOOLS_VERSION} \
+	&& ./configure \
+	&& make \
+	&& make install \
+	&& make clean
+
+
+# install vcftools
+ENV VCFTOOLS_VERSION="0.1.16"
+RUN cd home/tools \
+    && wget https://github.com/vcftools/vcftools/releases/download/v${VCFTOOLS_VERSION}/vcftools-${VCFTOOLS_VERSION}.tar.gz \
+    && tar xf vcftools-${VCFTOOLS_VERSION}.tar.gz \
+    && rm vcftools-${VCFTOOLS_VERSION}.tar.gz \
+	&& cd vcftools-${VCFTOOLS_VERSION} \
+	&& ./configure \
+	&& make \
+	&& make install \
+	&& make clean
+
+# install picard
+ENV PICARD_VERSION="2.25.0"
+RUN cd home/tools \
+    && wget https://github.com/broadinstitute/picard/releases/download/${PICARD_VERSION}/picard.jar 
+
+# install GATK
+ ENV GATK_VERSION="4.2.0.0"
+ RUN cd home/tools \
+    && wget https://github.com/broadinstitute/gatk/releases/download/${GATK_VERSION}/gatk-${GATK_VERSION}.zip \
+    && unzip gatk-${GATK_VERSION}.zip \
+    && rm gatk-${GATK_VERSION}.zip 
+
+
+# Install python packages
+RUN apt-get update \
+	&& apt-get dist-upgrade -y \
+	&& apt-get install -y \
```

Comment:
Would add --no-install-recommends here and other places
(from tip 4 @ https://www.docker.com/blog/intro-guide-to-dockerfile-best-practices/)